scripts/review/build_openrv.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# Copyright (c) Contributors to the aswf-docker Project. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -ex
+
+mkdir openrv
+cd openrv
+git clone --recursive https://github.com/AcademySoftwareFoundation/OpenRV.git .
+
+
+# Take latest from main for now.
+
+# Qt files are under /tmp/qttemp because OpenRV compiles it own PySide2 and it
+# causes issues during the build when Qt is within system folder (e.g. /usr/local)
+export QT_HOME=/tmp/qttemp
```

Comment:
I'd like to be able to allow to build it from source or to use an external PySide. I think it will be more doable once OpenRV is ported to Qt6/PySide6/Shiboken6 and OpenSSL 3. 

However, I don't have a timeline and I can't give an assurance that this will be done right now.